internal/tableprinter/tableprinter.go

```diff
@@ -0,0 +1,43 @@
+// Package tableprinter provides behavior to write tabular data to a given
+// destination.
+package tableprinter
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+)
+
+const (
+	tabwriterMinWidth = 6
+	tabwriterWidth    = 4
+	tabwriterPadding  = 3
+	tabwriterPadChar  = ' '
+	tabwriterFlags    = tabwriter.FilterHTML
+)
+
+// NewTabWriter returns a tabwriter that transforms tabbed columns into aligned
+// text.
+func NewTabWriter(output io.Writer) *tabwriter.Writer {
```

Comment:
Maybe make this `NewTablePrinter` to match the module name, so it's easier to find? Also, this name ties it more closely than necessary to the underlying `text/tabwriter` package, I think.

Then again, should this be capitalized, or is it really just a private helper for `PrintTwoColumnTable`?

Reply:
I didn't use `NewTablePrinter` because the function returns a `tabwriter.Writer`, and I did not think it was worth the effort to create a new class called `TablePrinter`.

At the moment, yes the tabwriter only exists to print this table. I can make it private.